schema.json

```diff
@@ -259,6 +264,39 @@
         }
       }
     },
+    "certificates": {
```

Comment:
Rename `certificates` -> `certifications`

Reply:
What's the reasoning here?

Reply:
Disregard this. I spent some time researching the difference between the two and `certificates` is more broadly applicable.